src/services/completions.ts

```diff
@@ -564,7 +576,11 @@ namespace ts.Completions {
                 };
             }
 
-            return undefined;
+            return {
+                isMemberCompletion: false,
+                isNewIdentifierLocation: false,

```

Comment:
Why is "isNewIdentifierLocation" true if there are results, but false if not? What difference does this make? (Leave a comment to explain if there's a reason for it).